tools/pd-ctl/pdctl/ctl_test.go

```diff
@@ -0,0 +1,73 @@
+package pdctl
+
+import (
+	"github.com/spf13/cobra"
```

Comment:
please format the goimports.

Reply:
i use idea auto format，what format need？

Reply:
the format like this?
import (
	"testing"

	"github.com/spf13/cobra"
)